crates/swift-integration-tests/src/option.rs

```diff
@@ -41,6 +41,15 @@ mod ffi {
         fn field(&self) -> u8;
     }
 
+    extern "Rust" {
+        type OptTestOpaqueRustTypeRef;
+
+        #[swift_bridge(init)]
```

Comment:
Can remove this `#[swift_bridge(init)]` as per the feedback below.

Reply:
Done